festim/_version.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# file generated by setuptools_scm
+# don't change, don't track in version control
+__version__ = version = '0.1.dev2157+g5abf34d.d20221108'
+__version_tuple__ = version_tuple = (0, 1, 'dev2157', 'g5abf34d.d20221108')
```

Comment:
This file should not be tracked in version control. Can you please remove it?

Reply:
Removed!